nix/ext/scripts/update_versions_json.py

```
@@ -0,0 +1,90 @@
+#! /usr/bin/env nix-shell
```

Ruff (0.14.11)

Shebang should contain `python`, `pytest`, or `uv run`

(EXE003)